src/main/java/com/minecolonies/coremod/MineColonies.java

```diff
@@ -96,6 +122,122 @@ public MineColonies()
 
         InteractionValidatorInitializer.init();
         proxy.setupApi();
+
+        IEventBus modEventBus = FMLJavaModLoadingContext.get().getModEventBus();
```

Comment:
I'd like to move this into its own dedicated worldgen event handler so its in line with most of our existing handlers